conans/client/generators/qbs.py

```diff
@@ -3,8 +3,7 @@
 
 
 class DepsCppQbs(object):
-    def __init__(self, deps_cpp_info, cpp_info):
-        super(DepsCppQbs, self).__init__(deps_cpp_info, cpp_info)
+    def __init__(self, deps_cpp_info):
```

Comment:
Thanks very much for this pull request! I think that you need the cpp_info as parameter, even if not used, as it will be instantiated with 2 params. It seems that this generator is nor properly covered by tests, so that is why the tests pass. Could you please fix the constructor? And I'll merge it.

BTW, if you don't know about generator packages, please check them in the docs, or ask me, they are really useful to implement and evolve generators quickly without depending on the conan app to be updated.

Thanks very much again!